src/views/partials/blogs/create.hbs

```diff
@@ -1,3 +1,6 @@
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/simplemde/latest/simplemde.min.css">
+<script src="https://cdn.jsdelivr.net/simplemde/latest/simplemde.min.js"></script>
```

Comment:
```suggestion
<link rel="stylesheet" href="https://unpkg.com/simplemde@1.11.2/dist/simplemde.min.css">
<script src="https://unpkg.com/simplemde@1.11.2/dist/simplemde.min.js"></script>
```
I think it is better to use unpkg for all of our static files: less dns lookup = a bit faster website!

Reply:
Good call, thanks for the feedback 👍